external-lb.go

```diff
@@ -77,8 +76,8 @@ func updateExistingConfigs(metadataConfigs map[string]model.LBConfig, providerCo
 			mLBConfig := metadataConfigs[key]
 			pLBConfig := providerConfigs[key]
 			var update bool
-			//check that the targetName and targets match
-			if mLBConfig.LBTargetName == pLBConfig.LBTargetName {
+			//check that the targetPoolName and targets match
+			if mLBConfig.LBTargetPoolName == pLBConfig.LBTargetPoolName {
```

Comment:
It is a check to make sure that we are updating a rancher added target pool - just being cautious incase there are out of band changes made on f5 to the virtualserver config